app/controllers/tickets_controller.rb

```diff
@@ -189,4 +188,25 @@ def anonymize
       new_wca_id: person&.reload&.wca_id, # Reload to get the new wca_id
     }
   end
+
+  private def ticket_and_acting_stakeholder_from_params
+    ticket = Ticket.find(params.require(:ticket_id))
+    acting_stakeholder = TicketStakeholder.find(params.require(:acting_stakeholder_id))
+
+    [ticket, acting_stakeholder]
+  end
+
+  private def check_ticket_errors(ticket, acting_stakeholder, action)
```

Comment:
Same thing about a pseudo-boolean being returned, which should be converted to `before_action` logic.